api/bases/dataplane.openstack.org_openstackdataplanenodes.yaml

```diff
@@ -106,6 +106,8 @@ spec:
                           type: string
                       type: object
                     type: array
+                required:
+                - ansibleSSHPrivateKeySecret
```

Comment:
Agreed, the SSH key is required to create data-plane{,role,node} objects. Else, we can't really manage them.